src/api/worker/search/Indexer.ts

```diff
@@ -170,10 +166,11 @@ export class Indexer {
 	/**
 	 * Opens a new DbFacade and initializes the metadata if it is not there yet
 	 */
-	async init({ user, userGroupKey, retryOnError, cacheInfo }: IndexerInitParams): Promise<void> {
+	async init({ user, userFacade, entityClient, retryOnError, cacheInfo }: IndexerInitParams): Promise<void> {
```

Comment:
idk if it makes sense to put both `userFacade` and `entityClient` here. It also doesn't care? Could we perhaps inject something that gives you a fresh key or a key of certain version? An object or just a function.

otherwise, you can just put these two in constructor but I honestly like it less